bigframes/session/clients.py

```diff
@@ -28,12 +28,13 @@
 import google.cloud.bigquery_storage_v1
 import google.cloud.functions_v2
 import google.cloud.resourcemanager_v3
+import ibis
 import pydata_google_auth
 
 import bigframes.version
 
 _ENV_DEFAULT_PROJECT = "GOOGLE_CLOUD_PROJECT"
-_APPLICATION_NAME = f"bigframes/{bigframes.version.__version__}"
+_APPLICATION_NAME = f"bigframes/{bigframes.version.__version__}/ibis/{ibis.__version__}"
```

Comment:
Needs to be space separated please.

```suggestion
_APPLICATION_NAME = f"bigframes/{bigframes.version.__version__} ibis/{ibis.__version__}"
```